keanu-project/src/main/java/io/improbable/keanu/util/io/MIRSaver.java

```diff
@@ -0,0 +1,70 @@
+package io.improbable.keanu.util.io;
+
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.mir.MIR;
+import io.improbable.mir.SavedBayesNet;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MIRSaver extends ProtobufSaver {
+
+    public final static String MODEL_NAME = "Saved Keanu Graph";
+    public final static String ENTRY_POINT_NAME = "Keanu Graph";
```

Comment:
Could we call the entry point 'Bayesian Network' or something else that's not Keanu specific?

Reply:
As above - I'd prefer for this to be Keanu specific - it is a graph that Keanu has saved and loaded after all.